### DISCUSSION TITLE
Appropriate Filter profile shape for Radio data (NOEMA/PdBI/VLA)

### DISCUSSION THREAD

**Discussion by ScorP1onnn** [mboquien/cigale#109] [Q&A]
Hello all,

I am trying to use the latest version of Cigale to fit data from the optical to the radio. In the radio range, I am using continuum estimates observed with NOEMA, PdBI, and some older data from VLA and MAMBO-I, such as in Omont et al. (2001) in A&A 374, 371 and Carilli et al. (2001) in AJ 122, 1679 (Carilli et al. (2001) observed with VLA 'A' configuration, band unspecified). However, Cigale does not have specific filters for these telescopes, as I believe that would depend on the particular observations made. 

Therefore, I would like to know how to create a filter for these observations. Can I construct a boxcar-shaped filter centered on the observed frequency of the continuum estimate (converting to wavelength for the filter) with a width similar to the bandwidth of the observations, such as approximately 8 GHz for the NOEMA/PdBI data? Additionally, could you guide me on how to approach this for MAMBO-I, considering that MAMBO-II does not follow a boxcar-shaped filter?

Thank you very much in advance,
ScorP1onnn

**Comment by mboquien** [comment]
You can create any filter you like. For instance look at the files in `database_builder/filters/`. You can find the `VLA_C` filter as an example for creating a new filter. The first column is the wavelength in Å and the second column is the transmission (arbitrary unit, CIGALE does its own normalization when importing the filter). The first three lines starting with `#` contain respectively the name of the filter, whether the filter is defined in terms of energy or photon, and the third line is a free comment to describe the filter. Then you can import it as `pcigale-filters add FILTERNAME`. You can check it has been correctly added with `pcigale-filters list`.

**Reply by ScorP1onnn** [reply]
Thank you very much for your quick response. I am familiar with how to create the filter, as it is well explained in the manual. However, I would like to confirm whether using a boxcar shape for the radio data, centered on the observed frequency of the continuum estimate, is the most appropriate filter shape for radio data or is there a different profile shape that might be more suitable that I may not be aware of?

**Reply by mboquien** [reply]
I think a square filter should be fine but best would be to contact the observatory to get their technical opinion on this.
